server/protocols/jmap-draft/src/main/java/org/apache/james/jmap/draft/model/message/view/MessageFastView.java

```diff
@@ -59,7 +59,7 @@
         protected Builder() {
             super();
             this.preview = Optional.empty();
-            this.hasAttachment = Optional.empty();
+            this.hasAttachment = Optional.of(false);
```

Comment:
The constructor is not the right place to pass a default value

Reply:
There are some fail tests if I don't put default value. `MessageFullViewTest.buildShouldWorkWhenMandatoryFieldsArePresent()` is an example

Reply:
If you have to position a default value, initialise to empty in the constructor and call .orElse(false) in the build method - that's for the "formalism" comment.

Reply:
I would be in favor of modifying such a test to make hasAttachment mandatory to specify.